.github/workflows/job_python_unit_tests.yml

```diff
@@ -257,18 +248,6 @@ jobs:
           export LD_LIBRARY_PATH=${INSTALL_TEST_DIR}:$LD_LIBRARY_PATH
           python3 ${OPENVINO_REPO}/docs/articles_en/assets/snippets/main.py
```

Comment:
maybe we can also include `Docs Python snippets` to `.github/workflows/job_python_api_tests.yml` ? As they test OpenVINO API code snippets